lib/dotcom/alerts/commuter_rail.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Dotcom.Alerts.CommuterRail do
+  @moduledoc """
+  Implements `Dotcom.Alerts.Group` for Commuter Rail alerts.
+  """
+
+  use Dotcom.Alerts.Group
+
+  import import Dotcom.Alerts, only: [service_impacting_effects: 0]
+
+  @impl Dotcom.Alerts.Group
+  def effects do
+    %{
+      "Bike" => [bike_issue: 1],
+      "Elevator & Escalator" => [
+        elevator_closure: 1,
+        escalator_closure: 1
+      ],
+      "Other" => [],
+      "Parking" => [parking_issue: 1],
+      "Service" => service_impacting_effects(),
+      "Track Change" => [track_change: 1]
+    }
+  end
+
+  @impl Dotcom.Alerts.Group
+  def group_order do
+    ["Service", "Track Change", "Elevator & Escalator", "Bike", "Parking", "Other"]
+  end
```

Comment:
Kinda like what your `group()` function outputs, I wonder if we should just write `effects` and `group_order` together in the first place.

```elixir
@impl Dotcom.Alerts.Group
def grouped_effects do
  [
     {"Service", service_impacting_effects()},
     {"Track Change", [track_change: 1]},
     {"Elevator & Escalator", [elevator_closure: 1, escalator_closure: 1],
     {"Bike", [bike_issue: 1]},
     {"Parking", [parking_issue: 1]},
     {"Other", []}
   ]
end
```

Then you'd only need to iterate through this one thing, it's already in order, etc

Reply:
I like that idea :)